.github/workflows/make-check.yml

```diff
@@ -11,8 +11,8 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macOS-13
-          - macOS-14
+          - macOS-15
+          - macOS-26
```

Comment:
Same suggestion as in https://github.com/crc-org/crc/pull/5011/commits/3391f98371dd082877ddc4ceaa0b85850fe32c7a#r2526670082